builder/tart/step_disk_file_prepare.go

```diff
@@ -44,6 +44,11 @@ func (s *stepDiskFilePrepare) Run(ctx context.Context, state multistep.StateBag)
 	partitionTable, err := disk.GetPartitionTable()
 
 	if err != nil {
+		if err.Error() == "unknown disk partition type" {
```

Comment:
I think we can terminate earlier (above) if we're dealing with a Linux VM because we only retrieve the partition table and modify it to remove the macOS recovery partition `RecoveryOSContainer`, which is never present on Linux.

Reply:
True, I guess we can use the presence of `fromISO` to skip, but if you just run an already installed Linux VM we don't have a way of knowing if it's Linux right now AFAICT. That's why I added the OS in https://github.com/cirruslabs/tart/pull/772 

Do you see a better heuristic to detect Linux?

Reply:
>but if you just run an already installed Linux VM we don't have a way of knowing if it's Linux right now AFAICT

Good point, I've overlooked that part.